source/administration/telemetry.rst

```diff
@@ -89,6 +89,7 @@ Non-personally Identifiable Diagnostic Information, distinguished by end users a
   - *Mobile Performance Diagnostics:* Load times for starting the app, switching channels, and switching teams 
   - *Permissions Discovery Diagnostics (Enterprise Edition Only):* Provides all the permissions configured for each role for the System Scheme and each Team Override Scheme created in the system.  Scheme ID; Team Admin Permissions; Team User Permissions; Channel Admin Permissions; Channel User Permissions; Number of teams the scheme is associated with
   - *Group Discovery Diagnostics (Enterprise Edition Only):* Provides information related to AD/LDAP groups, including number of groups synced to Mattermost, teams and channels associated to groups, teams and channels synced with groups, and number of group members
+  - *System Console Discovery Diagnostics:* click events on the hamburger menu and menu items: Administrator's Guide, Troubleshooting Forum, Commerical Support, About Mattermost; click events on the menu itesm in the left navigation.
```

Comment:
Is this for discovering the System Console? Or is this just for the menu items?

Reply:
```suggestion
  - *System Console Discovery Diagnostics:* click events on the hamburger menu and menu items: Administrator's Guide, Troubleshooting Forum, Commerical Support, About Mattermost; click events on the menu items in the left navigation.
```

Reply:
Just for the menu items

Reply:
Just minor tweak:

Reply:
```suggestion
  - *System Console Menu Discovery Diagnostics:* Clicks on the hamburger menu items of the System Console, including Administrator's Guide, Troubleshooting Forum, Commercial Support, About Mattermost, and clicks on the left-hand side navigation menu items
```